lib/tasks/export_users_auth0.rake

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+desc 'Export legacy users to Auth0'
+
+task 'export_users_to_auth0' => :environment do |_t, _args|
+  ActiveRecord::Base.logger.level = :info
+
+  options = { emails: [], export_all: false }
+
+  op = OptionParser.new do |o|
```

Comment:
nice to see this class... I haven't seen us use it in any other rake tasks 😄

Reply:
Yes, it is nice. The only issue with rake tasks and parameters is that you need to add two extra dashes (`--`) before the actual parameters, like in `rake my_task -- --my-parameter`, otherwise `rake` will think that is another task to be executed.